providers/base/tests/test_wwan_tests.py

```diff
@@ -0,0 +1,111 @@
+import unittest
+import sys
+from unittest.mock import patch, call, Mock, MagicMock
+from io import StringIO
+from contextlib import redirect_stdout
+
+# Mock the dbus module due to is is not available on CI testing environment
+sys.modules["dbus"] = MagicMock()
+
+import wwan_tests
+
+
+class TestMMDbus(unittest.TestCase):
+
+    @patch("wwan_tests.MMDbus.__init__", Mock(return_value=None))
```

Comment:
Is this Mock supposed to make the `obj_mmdbus = wwan_tests.MMDbus()` to get None?
Cause if so, then later on you're trying to assign to fields of None which would not work giving indication that the `__init__` is not actually mocked.

Reply:
No, this Mock is supposed to bypass to communicate with DBUS in the __init__ function.
So the `obj_mmdbus` will be a `MMDbus` object, not a None type variable.

Reply:
This is ok as `__init__` is not `__new__`